src/importexport/musicxml/internal/musicxml/import/importmusicxmlpass2.cpp

```diff
@@ -1299,6 +1301,7 @@ static void addMordentToChord(const Notation& notation, ChordRest* cr)
         if (color.isValid()) {
             mordent->setColor(color);
         }
+        mordent->setVisible(notation.visible());
```

Comment:
for consitency better move to above setting color?

Reply:
I like consitency